code/datums/traits/negative.dm

```diff
@@ -100,94 +100,27 @@
 	if(DT_PROB(0.05, delta_time))
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "depression", /datum/mood_event/depression)
 
+
 /datum/quirk/family_heirloom
 	name = "Family Heirloom"
 	desc = "You are the current owner of an heirloom, passed down for generations. You have to keep it safe!"
 	value = -1
 	mood_quirk = TRUE
 	var/obj/item/heirloom
 	var/where
+	var/static/list/random_bedsheets = subtypesof(/obj/item/bedsheet)
```

Comment:
There's a `/obj/item/bedsheet/random`, which you might be able to use here. If it's not suitable for some reason, it should likely be excluded from this list.

Reply:
`/obj/item/bedsheet` instead of `/obj/item/bedsheet/random` right? fixed. thank you!